keras_nlp/layers/transformer_decoder.py

```diff
@@ -84,6 +91,8 @@ def __init__(
         self.activation = activation
         self.layer_norm_epsilon = layer_norm_epsilon
         self._built = False
```

Comment:
Please move this line after 
```        
self.kernel_initializer = kernel_initializer
self.bias_initializer = bias_initializer
```
we want to keep attributes setting from constructor arguments together.

Reply:
Thanks for the review
Could you elaborate a bit which line should I move? Is this the `self._built = False` line?